scripts/provisioning-tests

```diff
@@ -55,7 +55,7 @@ export SOME_K8S_VERSION=$(curl -sS https://raw.githubusercontent.com/rancher/kon
 fi
 
 if [ "${DIST}" = "rke2" ]; then
-  export SOME_K8S_VERSION=v1.26.7+rke2r1
+  export SOME_K8S_VERSION=v1.27.10+rke2r1
```

Comment:
This version was pinned due to some flaky tests - https://github.com/rancher/rancher/pull/42796
Suggesting to either leave it untouched or remove this if statement altogether as it's not present in `release/v2.8` or `release/v2.9` lines so it looks like the issue with flaky test was addressed.

Not sure if it required any improvements in the tests themselves so if we want to err on the side of caution I would leave this untouched. @Oats87 / @jakefhyde / @HarrisonWAffel , please chime in.

Reply:
It's likely safe at this point to simply remove this pinning... if it passes, it passes, if not, then re-pin it?